articles/openshift/howto-deploy-java-eap-app.md

```diff
@@ -0,0 +1,384 @@
+
+5. Since our application uses Microprofile capabilities, you have to select for this demo the one named EAP Xp (at the time of this writing, the exact version is EAP Xp 1.0.0). The `Xp` stands for Expansion Pack. With the JBoss Enterprise Application Platform expansion pack, developers can use Eclipse MicroProfile application programming interfaces (APIs) to build and deploy microservices-based applications.
+
+6. Click on the `EAP Xp` Helm Chart and then `Install Helm Chart` button.
+
+At this point, we need to configure the chart to be able to build and deploy the application:
+
+1. Change the name of the release to `eap-todo-list-demo`. 
+1. We can configure the Helm Chart either using a `Form View` or a `YAML View`. Check `YAML View` radio button in the `Configure via` box.
+1. Then, change the YAML content to configure the Helm Chart by copying the content of the Helm Chart file available at `todo-list/deployment/application/todo-list-helm-chart.yaml` instead of the existing content:
+
+   ![OpenShift console EAP Helm Chart YAML content](./media/howto-deploy-java-eap-app/console_eap_helm_charts_yalm_content.png)
+
+1. Finally, click on the `Install` button to start the installation. This will open the `Topology view` with a graphical representation of the Helm release (named eap-todo-list-demo) and its associated resources.
+
+    ![OpenShift console topology](./media/howto-deploy-java-eap-app/console_topology.png)
+
+    The Helm Release (abbreviated HR) is named `eap-todo-list-demo`. It includes a Deployment resource (abbreviated D) also named `eap-todo-list-demo`. 
+
+1. When build is finished (the bottom-left icon will display a green check) and the application is deployed (the circle outline is in dark blue), you can go to application the URL (using the top-right icon) from the route associated to the deployment.
+
+    ![OpenShift console open application](./media/howto-deploy-java-eap-app/console_open_application.png)
+
+1. The application is opened in your browser looking similar to the following image:
+
+    ![OpenShift application running](./media/howto-deploy-java-eap-app/application_running_openshift.png)
+
+1. The application shows you the name of the pod which has served the information. To verify the clustering capabilities you could add some Todos, change the filter to see all the records and mark some of them as completed. Then delete the pod name that appears on the application, and once deleted, create a new Todo on the same application window. You will see that the new Todo is added via an ajax request, and you will continue getting the same list. It means your view, which was stored on the HTTP Session, has not been reset to a new one when the response is served by a different pod, indeed you will see that the session ID has not changed. However, this time the response was served by a different pod. That verifies that your HTTP Session data is being replicated across all your pods.
```

Comment:
Previously Ajax was capitalized but in this paragraph it is not. I am not sure which way is correct.

Reply:
What is intended by "delete the pod name" - is this rather to `delete the pod with the name indicated in the "Server Host Name" field' - perhaps you would give some information on how to delete the pod or a reference?

Reply:
If I followed correctly: `s/same list/expected list, including the new Todo/`

Reply:
Done, I've revisited this paragraph describing better what is happening with the cluster.